index.html

```diff
@@ -33,18 +33,176 @@ <h1 class="presentation">I'm Mehdi jdir
                 <h3>LET'S CONNECT</h3>
                 <div id="contact">
                    <ul>
-                     <li class="icon"><a href="" target="_blank"><img src="img/twitter.png" alt="twitter link"></a></li>
-                     <li class="icon"><a href="" target="_blank"><img src="img/Linkedin.png" alt="Linkedin link"></a></li>
-                     <li class="icon"><a href="" target="_blank"><img src="img/Mobile.png" alt="Mobile link"></a></li>
-                     <li class="icon"><a href="" target="_blank"><img src="img/github.png" alt="github link"></a></li>
-                     <li class="icon"><a href="" target="_blank"><img src="img/succes.png" alt="succes link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="./img/twitter.png" alt="twitter link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="./img/Linkedin.png" alt="Linkedin link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="./img/Mobile.png" alt="Mobile link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="./img/github.png" alt="github link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="./img/succes.png" alt="succes link"></a></li>
                                          
                    </ul>
                 </div>
             
         </section>
       </div>
+      <section class="gridsec">
+      <div class="page1">
+        <header>
+          <div class="image1"><img src="./img/Snapshoot Portfolio.png" alt="Snapshoot image1"</div>
+        </header>
```

Comment:
- [ ] Header tag? no please, you had a good idea but you did not well express it in your code, see? Since the image is wrapped in a div already, I think I should remove this header tag as it is not necessary and keep the div
- the header has its own purpose and place of use, please read this [here](https://www.w3schools.com/tags/tag_header.asp)